website/templates/account/login.html

```diff
@@ -18,18 +18,18 @@
           <a href="{% url 'index' %}" class="hover:no-underline text-white hover:text-white">{% env 'PROJECT_NAME' %}</a>
         </div>
         <p class="mt-6 font-normal text-center text-white text-xl md:mt-0">
-            OWASP BLT is a bug logging tool to report issues and get points, companies are held accountable.
+          OWASP BLT {% trans 'is a bug logging tool to report issues and get points, companies are held accountable' %}.
```

Comment:
here is another spot where OWASP BLT should be replaced with the {% PROJECT_NAME %} so in general - all areas where it says "Bugheist" or "OWASP BLT" should be  {% PROJECT_NAME %}